Form-Controls/index.html

```diff
@@ -13,15 +13,47 @@
     <h1>Product Pick</h1>
   </header>
   <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+  <form id="orderForm" onsubmit="return validateForm()">
+    <label for="name">Full Name:</label>
+    <input type="text" id="name" name="name" pattern="[A-Za-z]{2-32} {-}" required>
```

Comment:
May I know what is the `{ - }` in the pattern?

Reply:
Actually it was an accidental typo character, so I removed it.